src/pages/mining.md

```diff
@@ -126,6 +126,108 @@ In case you are running into issues or would like to see verbose logging, you ca
 BLOCKSTACK_DEBUG=1 stacks-node krypton
 ```
 
+## Running a miner in Windows
+
+### Generate keychain and get testnet tokens in Windows
+
+To setup the miner, first, we need to generate a keychain. With this keychain, we'll get some testnet BTC from a faucet, and then use that BTC to start mining.
+
+To get a keychain, the simplest way is to use the `stacks-cli`.We'll use the `stx make-keychain` command,and pass -t to indicate that we want a testnet keychain.
+
+Generate a keychain:
+
+```bash
+npm install --global @stacks/cli
+stx make_keychain -t > cli_keychain.json
+type cli_keychain.json
+```
+
+After this runs, you'll probably see some installation logs, and at the end you should see some JSON that looks like this:
+
+```json
+{
+  "mnemonic": "exhaust spin topic distance hole december impulse gate century absent breeze ostrich armed clerk oak peace want scrap auction sniff cradle siren blur blur",
+  "keyInfo": {
+    "privateKey": "2033269b55026ff2eddaf06d2e56938f7fd8e9d697af8fe0f857bb5962894d5801",
+    "address": "STTX57EGWW058FZ6WG3WS2YRBQ8HDFGBKEFBNXTF",
+    "btcAddress": "mkRYR7KkPB1wjxNjVz3HByqAvVz8c4B6ND",
+    "index": 0
+  }
+}
+```
+
+Request BTC from faucet:
+
+We need to get some testnet BTC to that address. Grab the btcAddress field, and call the BTC faucet:
+
+```bash
+# replace <btc_address> with `btcAddress` property from your keychain
+curl -XPOST "https://stacks-node-api.blockstack.org/extended/v1/faucets/btc?address=<btc_address>"
+```
+
+You'll be sent 0.5 testnet BTC to that address. Don't lose this information - we'll need to use the `privateKey` field later on.
+
+### Create configuration file
+
+Now, we need to configure our node to use this Bitcoin keychain. In the `folder where your binary is extracted`, create a new file called `testnet-miner-conf.toml`.
```

Comment:
`folder where your binary is extracted` > folder where your binary is extracted

(we should use `var` for code/var highlights mostly)

Reply:
Fixed it.